tests/driver_ccs811/main.c

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright (C) 2018 Gunar Schorcht
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     tests
+ * @brief       Test application for the Sensirion SHT30/SHT31/SHT35 device driver
+ * @author      Gunar Schorcht <gunar@schorcht.net>
+ * @file
+ *
+ * The test application demonstrates the use of the CCS811. It uses the default
+ * configuration parameters, that is, the measurement mode #CCS811_MODE_1S with
+ * one measurement per second.
+ *
+ * The application can use both approaches to wait for new data
+ *
+ * - using the data-ready interrupt #CCS811_INT_DATA_READY or
+ * - using the data-ready status function #ccs811_data_ready
+ *
+ * To use the data-ready interrupt, use module ```ccs811_full``` and
+ * define ```USE_CCS811_DATA_READY_INT``` in CFLAGS variable of the
+ * make command, for example:
+ *
+ * ```
+ * USEMODULE=ccs811_full CFLAGS="-DUSE_CCS811_DATA_READY_INT" \
+ * make flash BOARD=... -C tests/driver_ccs811
+ * ```
+ */
+
+#include <stdio.h>
+#include <string.h>
+
+#include "thread.h"
+#include "xtimer.h"
+
+#include "ccs811.h"
+#include "ccs811_params.h"
+
+#if USE_CCS811_DATA_READY_INT && !MODULE_CCS811_FULL
```

Comment:
Now that I look at it, I think there should be a test application for the simple case and another for the full featured sensor.
In the current state, the functions of the full featured sensor are not built automatically by the CI.

Reply:
> Would it be enough to have tests/driver_ccs811 and tests/driver_ccs811_full?Should the latter one have interrupts enabled by default?

Yes to both and no need to keep interrupt code in the "polling" application, just a reference in the README would be enough I think.
Like it now the _full_ version code is not built by the CI (since it is skipped by the preprocessor macros).